Directory.Build.targets

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <ItemGroup>
+    <PackageReference Update="McMaster.Extensions.CommandLineUtils" Version="2.3.4" />
+    <PackageReference Update="Microsoft.Build.Utilities.Core" Version="15.5.180"/>
+    <PackageReference Update="Microsoft.CodeAnalysis.CSharp" Version="2.10.0" />
+    <PackageReference Update="Microsoft.Extensions.FileSystemGlobbing" Version="2.0.1" /> 
+    <PackageReference Update="Microsoft.NET.Test.Sdk" Version="15.9.0" /> 
+    <PackageReference Update="Microsoft.TestPlatform.ObjectModel" Version="16.1.0" />
+    <PackageReference Update="Mono.Cecil" Version="0.10.4" />
+    <PackageReference Update="Moq" Version="4.10.1" />
+    <PackageReference Update="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Update="System.Reflection.Metadata" Version="1.5.0" />
+    <PackageReference Update="xunit" Version="2.4.1" />
+    <PackageReference Update="xunit.runner.visualstudio" Version="2.4.1"/>
+    <DotNetCliToolReference Update="dotnet-xunit" Version="2.3.1" />
```

Comment:
@tonerdo @MarcoRossignoli dotnet-xunit has been deprecated for over a year now. I highly suggest to switch to dotnet test instead.

Reply:
Never used, @daveMueller can you remove this line?

Reply:
Removed it.